lib/gui/os/window-progress.js

```diff
@@ -18,6 +18,40 @@
 
 const electron = require('electron');
 const utils = require('../../shared/utils');
+const progressStatus = require('../modules/progress-status');
+
+/**
+ * @summary The title of the main window upon program launch
+ * @type {String}
+ * @private
+ * @constant
+ */
+const initialTitle = document.title;
+
+/**
+ * @summary Make the full window status title
+ * @private
+ *
+ * @param {String} baseTitle - base window title
+ * @param {Object} currentFlashState - flash state object
+ *
+ * @returns {String}
+ *
+ * @example
+ * const title = makeTitle({
+ *   type: 'write',
+ *   percentage: 55,
+ *   speed: 2049
+ * });
+ *
+ * console.log(title);
+ * // 'Etcher \u2013 55% Flashing'
+ */
+const makeTitle = (baseTitle, currentFlashState) => {
```

Comment:
Like with `progressStatus.fromFlashState`, I wonder if it makes sense to rename this `currentFlashState` parameter to just `state`?

And a minor nitpick, but the jsdoc for `progressStatus.fromFlashState` refers to `flashing metadata` but the jsdoc here refers to `flash state object`. AFAICT these are actually the same thing, so a bit of consistency might be nice ;)